docs/heroku-deployment.md

```diff
@@ -0,0 +1,19 @@
+
+Step by step guide on how to deploy the app in Heroku and enable automatic pull request deployment as well.
```

Comment:
```suggestion
# Setting up deployment in Heroku and automated pull request deployment
```

Reply:
Removing blank line from the start of the file and converting the first sentence into a header